src/scss/_customs.scss

```diff
@@ -0,0 +1,19 @@
+//Negative Margins
+$enable-negative-margins: true;
+$spacer: 1rem;
+$spacers: (
+  0: 0,
+  1: $spacer * .25,
+  2: $spacer * .5,
+  3: $spacer,
+  4: $spacer * 1.5,
+  5: $spacer * 3,
+  6: $spacer * 6,
+  7: $spacer * 9,
+);
+
+//Links
+$link-decoration: none;
+$link-hover-decoration: italic;
+
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
```

Comment:
To follow the convention of clean code, please add an empty line at the bottom of the file.  

- This visually separates the document's end from the file's end, aiding navigation and comprehension.

- Compatibility: Some older systems may struggle to parse files lacking a newline character at the end, causing potential issues.

- Concatenation: Ensures proper separation of concatenated files, preventing unexpected behavior in the output.

Please apply to all files.

Reply:
applied!